modules/react/toast/stories/examples/RTL.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import Toast from '../../index';
+import {ContentDirection, StaticStates} from '../../../common';
```

Comment:
Let's update this to use the package so it's easier to copy / paste as an example. I see the other `Toast` examples also have relative imports though.

`StaticStates` is really intended for our visual regression tests. [Here's an example](https://workday.github.io/canvas-kit/?path=/story/testing-react-popups-toast--toast-states). Let's make that `CanvasProvider` so it's closer to how someone would use it in their application.

```suggestion
import {ContentDirection, CanvasProvider} from '@workday/canvas-kit-react/common';
```